src/TextForEvent.tsx

```diff
@@ -498,10 +498,10 @@ function textForPowerEvent(event: MatrixEvent): () => string | null {
         if (!Number.isInteger(to)) {
             to = currentUserDefault;
         }
+        const displayName = MatrixClientPeg.get().getUser(userId).displayName;
```

Comment:
Displaynames are per-room, not global, so you can't use the displayName of getUser in a room. You'll need to get the room member instead